pages.de/common/nix-shell.md

```diff
@@ -0,0 +1,28 @@
+# nix-shell
+
+> Startet eine interaktive Shell, die auf einem Nix-Ausdruck basiert.
+> Weitere Informationen: <https://nixos.org/manual/nix/stable/#sec-nix-shell>.
+
+- Start mit Nix-Ausdruck in `shell.nix` oder `default.nix` im aktuellen Verzeichnis:
+
+`nix-shell`
+
+- Shell-Befehl in nicht-interaktiver Shell ausführen und beenden:
+
+`nix-shell --run "{{Befehl}} {{Befehl_Argumente}}"`
+
+- Start mit Ausdruck in `default.nix` im aktuellen Verzeichnis:
+
+`nix-shell {{default.nix}}`
+
+- Start mit aus nixpkgs geladenen Paketen:
+
+`nix-shell --packages {{Paket_name_1}} {{Paket_name_2}}`
```

Comment:
The following fix could be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/nix-shell.md) page is updated:

```suggestion
`nix-shell --packages {{Paket_name_1 Paket_name_2 ...}}`
```

Wrong syntax is used for zero or more arguments.

Reply:
Same here
```suggestion
`nix-shell --packages {{paket_name_1 paket_name_2 ...}}`
```